module/hooks/chat.mjs

```diff
@@ -117,10 +117,21 @@ function triggerRollStep(argString) {
 
 function addUserPortrait( msg, jquery ) {
```

Comment:
If the `isGM` wouldn't be necessary, then the whole block could be streamlined, I think, if so wanted:

```javascript
avatar = chatAvatarSetting === "selectedToken"
    ? canvas.tokens.controlled[0]?.document.texture.src
    : msg.user.avatar;
```

Possible since the assignment in the first `if` is the same as the nested `else`.